frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -118,8 +119,13 @@ func (a *CreateCVDAction) launchCVD(op apiv1.Operation) {
 	}
 }
 
+const userArtifactPrefix = "@user_artifact/"
+
 func (a *CreateCVDAction) launchWithCanonicalConfig(op apiv1.Operation) (*apiv1.CreateCVDResponse, error) {
 	data, err := json.MarshalIndent(a.req.EnvConfig, "", " ")
+	data = bytes.ReplaceAll(data,
```

Comment:
let's move this after the error handling: `if err != nil { ...` lines

Reply:
Moved ReplaceAll(...) code after error handling.